src/shell/plugins/fence.c

```diff
@@ -163,7 +165,8 @@ static void fence_shell_cb (const flux_msg_t *msg, void *arg)
         goto error;
     }
     if (fx->trace_flag)
-        shell_trace ("starting pmix exchange");
+        shell_trace ("starting pmix exchange: size %zi",
+                     codec_data_length (xdata));
```

Comment:
Ah, the coded_data_length is only used if a trace flag is set, so probably not a big deal.

Reply:
Just for full disclosure, the trace flag is currently stuck on.  My plan was to tie that to some config option later on.